tests/extproc/testupstream_test.go

```diff
@@ -217,6 +217,18 @@ data: [DONE]
 			responseBody:    `{"message": "aws bedrock rate limit exceeded"}`,
 			expResponseBody: `{"type":"error","error":{"type":"ThrottledException","code":"429","message":"aws bedrock rate limit exceeded"}}`,
 		},
+		{
+			name:            "forbidden",
+			backend:         "aws-bedrock",
+			path:            "/v1/chat/completions",
+			method:          http.MethodPost,
+			requestBody:     `{"model":"something","messages":[{"role":"system","content":"You are a chatbot."}]}`,
+			responseBody:    `{"type": "error", "error": {"type": "AccessDeniedException:http://internal.amazon.com/coral/com.amazon.bedrock/", "code": "403", "message": "You don't have access to the model with the specified model ID."}}`,
+			expPath:         "/model/something/converse-stream",
```

Comment:
```suggestion
			expPath:         "/model/something/converse",
```

sorry stream was not necessary